startup_jpoode.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# NOTE: This is a bash script intended to be used by the Dockerfile and not intended to be used on its own
+
+# Create the necessary openrc directories on boot of the new container
+sh -c rc-status
+# Create the necessary file to run a service on openrc. This is necessary when using an alpine image
+touch /run/openrc/softlevel
+# Start the SSH/SCP server on boot of the new container
+rc-service sshd start
```

Comment:
What password can be used to SSH into the container after spinning it up locally?

Reply:
It shouldn't require one with a properly created SSH key that can be applied to the authorized_keys through either a local volume or an attached volume in K8s. I can specify this in the documentation.

Reply:
I have added a new comment line to explain this a bit. I have also included how to add this authorized_keys through a mounted volume. Once again, though, I only recommend using this option in K8s. Having an unexposed SSH server isn't going to cause any issues, however.